__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -143,7 +147,8 @@ describe('TESTING ChatSchema', () => {
         message: {
           created_at: 'createdAt',
           id: 'messageId',
-          updated_at: 'updatedAt'
+          updated_at: 'updatedAt',
+          attachments: '[]'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` field is initialized as a string representation of an empty array. This is correct if we are storing JSON strings in the database. However, when reading from the database, you should parse this JSON string back into an array for use in your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->